tests/run.sh

```diff
@@ -92,10 +93,12 @@ cleanup() {
         kill -TERM "$p" 2>/dev/null || true
     done
     rm -rf "$tmp_files" || true
+    rm -rf 
```

Comment:
oh no, I added a thing which I then forgot about when I realized I could use the `tmp_files` thing - and I forgot to remove this half-implemented stub